File: auth/src/main/java/com/diagorn/sparkathon/auth/dto/user/RoleDTO.java
package com.diagorn.sparkathon.auth.dto.user;

import io.swagger.v3.oas.annotations.media.Schema;
import lombok.Data;

/**
 * Role DTO
 *
 * @author diagorn
 */
@Data
public class RoleDTO {
    /**
     * Role ID
     */
    @Schema(description = "Role ID", example = "1")
    private Long id;
    /**
     * Role name
     */
    @Schema(description = "Role name", example = "PARTICIPANT")
    private String name;
}

File: auth/src/main/java/com/diagorn/sparkathon/auth/dto/user/RoleDto.kt
package com.diagorn.sparkathon.auth.dto.user

import io.swagger.v3.oas.annotations.media.Schema

/**
 * Role DTO
 *
 * @author diagorn
 */
class RoleDto (
    /**
     * Role ID
     */
    @Schema(description = "Role ID", example = "1")
    val id: Long,

    /**
     * Role name
     */
    @Schema(description = "Role name", example = "PARTICIPANT")
    val name: String
)